config/routes.rb

```diff
@@ -1,5 +1,10 @@
+require 'sidekiq/web'
+
 Rails.application.routes.draw do
   mount ActionCable.server => '/cable'
+  authenticate :user, ->(user) { user.admin? } do
+    mount Sidekiq::Web => '/sidekiq'
```

Comment:
```suggestion
    mount Sidekiq::Web => '/jobs'
```

Personal preference. Allows you to later change the UI/backing job system without changing the URL :D

Reply:
I'm ok with this for now, as it lets me try multiple backends in parallel. But I might shift before launch. Thanks for the suggestion.